tests/acceptance/builds/prioritize-test.js

```diff
@@ -0,0 +1,84 @@
+import { module, test } from 'qunit';
+import { visit, click, triggerKeyEvent } from '@ember/test-helpers';
+import { setupApplicationTest } from 'travis/tests/helpers/setup-application-test';
+import signInUser from 'travis/tests/helpers/sign-in-user';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+module('Acceptance | builds/prioritize', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    const currentUser = this.server.create('user');
+    signInUser(currentUser);
+  });
+
+  test('prioritizing build button does not exsist if the build is not having permission to prioritize becuase of the organization', async function (assert) {
```

Comment:
Just a little typo:
```suggestion
  test('prioritizing build button does not exsist if the build is not having permission to prioritize because of the organization', async function (assert) {
```